packager/packager.js

```diff
@@ -136,7 +136,7 @@ function getDevToolsLauncher(options) {
       var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
       var script = 'launchChromeDevTools.applescript';
       console.log('Launching Dev Tools...');
-      exec(path.join(__dirname, script) + ' ' + debuggerURL, function(err, stdout, stderr) {
+      exec('"' + path.join(__dirname, script) + '" ' + debuggerURL, function(err, stdout, stderr) {
```

Comment:
I think this can be

```
child_process.execFile(path.join(__dirname, script), [debuggerURL], function...
```

which avoids the need to quote and is safer.